react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -5754,7 +5767,7 @@ exports[`Spinner should render examples: Spinner 5`] = `
     <div class=\\"styles__c-spinner___1snK7\\"><svg class=\\"styles__icon___23x3R styles__icon--spin___ybfC1\\" style=\\"fill: var(--primaryColor);\\" width=\\"80\\" height=\\"80\\">
         <use xlink:href=\\"#spinner\\"></use>
       </svg>
-      <p>loading.hi</p>
+      <p>loading.helloWorld</p>
```

Comment:
We should have `Hello World !` with your modification, no? (https://github.com/cozy/cozy-ui/pull/1319/files#diff-aae4f4155d6342669aceedd221607d8eR58)

Reply:
Yes, this is weird, the Argos screenshot had the change. Looking into it.

Reply:
https://github.com/cozy/cozy-ui/pull/1325, thanks good catch